var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -74,4 +83,13 @@ def cmake_args(self):
             '-Wno-dev',
             '-DBUILD_TESTING={0}'.format('ON' if self.run_tests else 'OFF')
         ])
+        if '~cuda' in self.spec:
+            options.extend(['-DUSE_CUDA=OFF'])
         return options
+
+    @run_after('build')
+    @on_package_attributes(run_tests=True)
```

Comment:
These two lines aren't needed since you gave the function the same name as the testing method in the base class

Reply:
Got it!  I copied that block from another package in spack (I forget which one).  Removing now...

Reply:
It's needed for packages like `perl` because the `Package` base class doesn't define any test phases, but not for packages that extend `CMakePackage` or most other popular build systems.